### PR TITLE
Remove password prompt when uploading .cer certificate

### PR DESCRIPTION
**Pull request by Mitko-Kerezov**
Those come from Apple and don't have a password
Fixes http://teampulse.telerik.com/view#item/273404
Ping @rosen-vladimirov @Fatme @teobugslayer

### CODE REVIEW

**Comment by rosen-vladimirov** [comment]
:+1:  after green build

File: lib/commands/cryptographic-identities.ts [hunk 1]
```diff
@@ -590,7 +590,7 @@ export class ImportCryptographicIdentity implements ICommand {
 				this.$errors.fail("The file '%s' does not exist.", certificateFile);
 			}
 
-			if(!password) {
+			if(!password && importType === "Pkcs12") {
```
Comment (Fatme) [inline lib/commands/cryptographic-identities.ts]: I suggest you to extract `Pkcs12` as a constant. We use it more than one time.

**Comment by Mitko-Kerezov** [comment]
Addressed comment by @Fatme

**Comment by Fatme** [comment]
:+1:

**Pull request merged**